foundation/docs/blog/posts/2024-conference-recap.md

```diff
@@ -0,0 +1,303 @@
+---
+title: 2024 Conference Recap
+date: 2024-04-28
+categories: 
+    - Conference Updates
+authors:
+    - masonegger
+---
+
+The annual PyTexas Conference was held at the Austin Central Library April 19 
+- 21, 2024 in Austin, TX. It was our largest conference yet, and we're excited
```

Comment:
This might render differently, but in the markdown preview this becomes a bulletpoint

Reply:
Oh yeah might need a newline before

Reply:
This is rendering fine on mkdocs. Hooray for different markdown processors.